cs/index.json

```diff
@@ -78,7 +78,8 @@
   },
   "main": {
     "error_support_64_os_only": "Podporujeme pouze 64bitové operační systémy",
-    "error_have_no_mining_device": "Kompatibilní zařízení pro těžbu nebylo nalezeno"
+    "error_have_no_mining_device": "Kompatibilní zařízení pro těžbu nebylo nalezeno",
+    "error_cannot_connect_pool": "Miner '{{miner}}' can not connect pool ({{poolinfo}}). Please check connection again"
```

Comment:
I think we need to indicate what the user needs to do in order to resolve this? Check firewall blocking connection? Check if the port is accessible?

Reply:
How about this?

"Miner '{{miner}}' can not connect pool ({{poolinfo}}). Please check your connection and firewall to make sure port {{port}} is not blocked"

Reply:
Sure - also not sure if we need the Miner '{{miner}}' 

Assuming remove Miner '{{miner}}' 
Chinese translation
```
無法連接礦池({{poolinfo}})。請檢查您的連接和防火牆以確保端口{{port}}未被阻擋。”
```

Reply:
because i should this message while benchmark too => When Crypto Miner benchmarks, there is a case that Crypto Miner run a lot of miner tools (dstm + ewbf) => if we have no miner type in message, user thinks it is the same error? or

Reply:
I don't think that'll be an issue since we shouldn't be trying the next miner in benchmark if this miner is not throwing error (in this case, it is a hard/manual stop)

Reply:
https://www.screencast.com/t/Dae6AM7j
This is the issue that is would like to prevent.  We have 2 same error messages as above because when we start miner (and the port is in use)
 - dstm => fail => show one message (fall back miner ...)
-  efbw => fail => show one message

Reply:
fixed as your suggestion